### PR TITLE
Introduce publication

### PR DESCRIPTION
**Pull request by daddykotex**
There!

I finally ran the tests (some weird sbt-junit-interface issue)...

Anyway the output of jvm 18 was a tidy bit different being 18.0.1

### CODE REVIEW

**Comment by daddykotex** [comment]
with your approval, I think this PR will be green @alexarchambault

File: build.sbt [hunk 1]
```diff
@@ -262,9 +263,7 @@ lazy val `interface-test` = project
     },
     Test / unmanagedClasspath ++= Def.taskDyn {
       if (sys.env.get("TEST_VERSION").isEmpty)
-        Def.task {
-          Seq((interface / finalPackageBin).value)
-	}
+        Def.task { Seq((interface / finalPackageBin).value) }
```
Comment (alexarchambault) [inline build.sbt:266]: Let's keep the former code here, it doesn't change anything.

**Review by alexarchambault — COMMENTED** [review]
Thanks @daddykotex! I only left minor comments.

File: build.sbt (hunk 1 above)
Comment (daddykotex) [reply build.sbt]: it's my mistake, I thought I was fixing bad idnentation, but turns out it was me that I had messed up the original code

**Comment by daddykotex** [comment]
should be ready to go

**Comment by daddykotex** [comment]
bumping so you can trigger a build so I can any remaining issue

**Review by alexarchambault — APPROVED** [review]
Merging, thanks again!

**Pull request merged**